import_install_example.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Library download function, optionally accepts a full path/name and URL
+function download_lib {
+    tmp_path="${1:-$(mktemp)}"
+    lib_url="${2:-https://raw.githubusercontent.com/sdelements/shtdlib/master/shtdlib.sh}"
+    curl -s -l -o "${tmp_path}" "${lib_url}" || wget --no-verbose "${lib_url}" --output-document "${tmp_path}" || return 1
+}
+
+# Library install function, optionallly accepts a URL and a full path/name
+# shellcheck disable=SC2120,SC2119
+function install_lib {
+    lib_path="${1:-/usr/local/bin/shtdlib.sh}"
+    lib_name="$(basename "${lib_path}")"
+    tmp_path="$(mktemp)"
+
+    echo "Installing library ${lib_name} to ${lib_path}"
+    original_umask="$(umask)"
+    umask 0022
+    download_lib "${tmp_path}"
```

Comment:
Add `... || return 1` if the call to `download_lib` fails. No point in continuing...

Reply:
Not true, we still want to fix the umask so I'm going to fix this using the umask decorator :)

Reply:
Good point